components/automate-ui/src/app/pages/user-management/user-management.component.ts

```diff
@@ -74,6 +72,17 @@ export class UserManagementComponent implements OnInit, OnDestroy {
         // Sort naturally first by name, then by id
         this.users = ChefSorters.naturalSort(users, ['name', 'id']);
       });
+
+    this.store.pipe(
```

Comment:
moved this subscription for handling create success into `ngOnInit`
right now we don't catch conflict errors in the user modal the way we do with teams and tokens.